atecc608a/main.c

```diff
@@ -76,10 +76,22 @@
     "the slot now behaves according to the policies set by the associated\n"\
     "configuration zone’s values. [y/n]: "
 
-/* Data used by tests */
+/* Physical locations of the keys in the secure element. */
 psa_key_slot_number_t atecc608a_private_key_slot = 1;
```

Comment:
Since we're separating slots from ID's - it would be good to use slot 0 as the private key, since it's an invalid key ID.

Reply:
Slot 0 is often used by Atmel for pre-provisioned keys (added by Atmel before they ship the parts out). For examples that generate new keys, I did not want to use that slot. The factory device provisioning example does use Slot 0 for a pre-provisioned key.